Makefile

```diff
@@ -25,7 +25,7 @@ all: test build image
 
 test:
 	@echo "Running the tests for $(IMAGE_NAME)..."
-	@pytest src/
+	@cd src/ && pytest; cd ..
```

Comment:
it was actually ok. also you don't have to `cd ..` in makefile because each line is run in a different shell

Reply:
:+1: fixed the pythonpath (failed to run locally)